pylint/test/functional/overlapping_exceptions.py

```diff
@@ -0,0 +1,35 @@
+# pylint: disable=missing-docstring
+
+class SomeException(Exception):
```

Comment:
We can try with some builtin exceptions as well, especially on Python 3, where IOError is now OSError. Check some other test files for seeing how to make a test only for a particular range of Python versions (there should be a corresponding .rc file containing the exclusions)